portal-api/src/v1/gef/models/facilities.js

```diff
@@ -177,7 +177,7 @@ class Facility {
         this.monthsOfCover = null;
       }
 
-      if (req.coverDateConfirmed) {
+      if (req.coverDateConfirmed === true || req.coverDateConfirmed === false) {
```

Comment:
what other value do we expect here? `null` ?

Reply:
Yep null